pkg/collector/corechecks/servicediscovery/module/comm_test.go

```diff
@@ -132,31 +133,20 @@ func TestShouldIgnoreComm(t *testing.T) {
 
 			require.EventuallyWithT(t, func(collect *assert.CollectT) {
 				proc, err := customNewProcess(int32(cmd.Process.Pid))
-				require.NoError(t, err)
+				assert.NoError(t, err)
 
 				ignore := discovery.shouldIgnoreComm(proc)
-				require.Equal(t, test.ignore, ignore)
+				assert.Equal(t, test.ignore, ignore)
 			}, 10*time.Second, 100*time.Millisecond)
```

Comment:
neat: consider reducing the timeout (10s) to something smaller (3s)

Reply:
reduced